src/Microsoft.AspNetCore.Mvc.Razor/TagHelperComponent/BodyHeadTagHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.TagHelperComponent
+{
+    /// <summary>
+    /// A <see cref="TagHelperComponentTagHelper"/> targeting &lt;body&gt; and &lt;head&gt; elements.
+    /// </summary>
+    [HtmlTargetElement("body")]
+    [HtmlTargetElement("head")]
+    public class BodyHeadTagHelper : TagHelperComponentTagHelper
```

Comment:
This should be two classes so people can toggle `head` and `body` `TagHelper`'s on/off as needed. Aka, if we have a `HeadTagHelper` and a `BodyTagHelper` a user can `@removeTagHelper Microsoft.AspNetCore.Mvc.Razor.BodyTagHelper, Microsoft.AspNetCore.Mvc.Razor` without also nuking the `head` capability.  Will need to have an additional `@addTagHelper` for the `HeadTagHelper`